packages/react-dom/src/events/plugins/__tests__/SimpleEventPlugin-test.js

```diff
@@ -208,7 +208,7 @@ describe('SimpleEventPlugin', function () {
             ref={el => (button = el)}
             onFocus={this.increment}
             onClick={() => {
-              // The focus call synchronously dispatches a nested event. All of
+              // The focus called synchronously dispatches a nested event. All of
```

Comment:
This is referencing the "call" to "focus" to it's correct as-is.

Reply:
Thank you for your patience in explaining and replying